src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4235,6 +4237,112 @@ const std::string ApplicationManagerImpl::AppHMITypeToString(
       return "PROJECTION";
     case mobile_apis::AppHMIType::REMOTE_CONTROL:
       return "REMOTE_CONTROL";
+    case mobile_apis::AppHMIType::WEB_VIEW:
+      return "WEB_VIEW";
+    default:
+      return "INVALID_ENUM";
+  }
+}
+
+mobile_apis::PredefinedLayout::eType
```

Comment:
@AByzhynar Could we consider another place for conversion functions?

Reply:
@mvorobio Propose one please. Also do you propose to refactor app manager in order to extract all conversion functions? We have many...

Reply:
@AByzhynar @mvorobio I think we can create something like app_mngr_helper here https://github.com/LuxoftSDL/sdl_core/tree/develop/src/components/application_manager/src/helpers and put these function there. Don't need to move all conversion functions, only the new one

Reply:
@LitvinenkoIra Rejected. If we go your proposed way, as a result, we will have 3 different places with conversion functions. At the moment I would like to keep them all in the one place for further refactoring.